src/realm/dictionary.cpp

```diff
@@ -369,6 +369,14 @@ void Dictionary::sort(std::vector<size_t>& indices, bool ascending) const
 }
 void Dictionary::distinct(std::vector<size_t>&, util::Optional<bool>) const {}
 
+Obj Dictionary::create_and_insert_linked_object(Mixed key)
```

Comment:
Should there be a version of this that can create an embedded object in a `Dictionary<Mixed>`?

Reply:
We don't support linking to objects in arbitrary tables.